std-bits/base/src/main/java/org/enso/base/XML_Utils.java

```diff
@@ -23,13 +23,16 @@ public class XML_Utils {
    * @throws IllegalAccessException if the DOM implementation class cannot be accessed.
    * @throws InstantiationException if the DOM implementation class cannot be instantiated.
    */
-  public static String outerXML(Node element)
+  public static String outerXML(Node element, boolean prettyPrint)
       throws ClassNotFoundException, IllegalAccessException, InstantiationException {
     DOMImplementationLS dom =
         (DOMImplementationLS) DOMImplementationRegistry.newInstance().getDOMImplementation("LS");
     LSSerializer serializer = dom.createLSSerializer();
     DOMConfiguration config = serializer.getDomConfig();
     config.setParameter("xml-declaration", false);
+    if (prettyPrint) {
+      config.setParameter("format-pretty-print", Boolean.TRUE);
```

Comment:
```suggestion
      config.setParameter("format-pretty-print", true);
```

Reply:
Pretty sure you could just pass the `prettyPrint` as well (regardless of value).